crates/forge/bin/cmd/getsolc.rs

```diff
@@ -0,0 +1,45 @@
+use clap::{Parser, ValueHint};
+use eyre::Result;
+use foundry_compilers::Graph;
+use foundry_config::Config;
+use std::path::PathBuf;
+use yansi::Paint;
+
+/// CLI arguments for `forge get-solc`.
+#[derive(Clone, Debug, Parser)]
+pub struct GetSolcArgs {
```

Comment:
can this be folded into another command? Feels a bit weird that's standalone but not a big deal.